notifikasjon/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/notifikasjon/OpprettOppgaveServiceTest.kt

```diff
@@ -36,73 +37,54 @@ class OpprettOppgaveServiceTest {
     }
 
     @Test
-    fun `erFeilMelding`() {
-        val fellesDeprecatedFail = Fail(
-            eventName = EventName.OPPGAVE_OPPRETT_REQUESTED,
-            behov = BehovType.OPPRETT_OPPGAVE,
-            feilmelding = "FEIL",
-            data = null,
-            uuid = UUID.randomUUID().toString(),
-            forespørselId = UUID.randomUUID().toString()
-        ).toJsonMessage()
-        assertEquals(Transaction.TERMINATE, service.determineTransactionState(fellesDeprecatedFail))
-        assertTrue(service.isFailMelding(fellesDeprecatedFail))
-    }
-
-    @Disabled("Enable om vi begynner å bruke ny Fail-objekt sammen med CompositeEventListener")
-    @Test
-    fun `feil fra behov tolkes som feil av service`() {
-        val failFraBehov = Behov.create(
+    fun `feil tolkes som feil av service`() {
+        val fail = Fail(
+            feilmelding = "Opprett oppgave feilet.",
             event = EventName.OPPGAVE_OPPRETT_REQUESTED,
-            behov = BehovType.OPPRETT_OPPGAVE,
-            forespoerselId = "987654321"
-        ).createFail("Uff")
-        assertTrue(service.isFailMelding(failFraBehov.jsonMessage))
-    }
+            transaksjonId = UUID.randomUUID(),
+            forespoerselId = UUID.randomUUID(),
+            utloesendeMelding = mapOf(
+                Key.BEHOV.toString() to BehovType.OPPRETT_OPPGAVE.toJson()
```

Comment:
Vil det bare være Behov i utloesendeMelding?

Reply:
I virkeligheten vil det være mer, men det vil ikke ha noe å si for testen. Jeg prøvde bare å lage noe så likt som den originale testen.